src/run.rs

```diff
@@ -50,8 +50,15 @@ pub async fn image_ids(docker: &Docker) -> io::Result<HashSet<String>> {
                 format!("Unable to determine IDs of all images: {:?}.", error),
             )
         })?;
+    let mut images = HashMap::new();
 
-    Ok(output.into_iter().map(|image| image.id).collect())
+    for image in output {
+        match image.created.try_into() {
+            Ok(unsigned) => images.insert(image.id, Duration::from_secs(unsigned)),
```

Comment:
This is totally subjective, but I think the `unsigned` variable could have a better name. Maybe `created` to match the name of where it comes from? (I'm not super fond of `created`, but consistency is more important than my personal preferences.)